src/components/DeliverablesTable/index.tsx

```diff
@@ -41,33 +44,50 @@ const DeliverablesTable = ({
 }: DeliverablesTableProps) => {
   const dispatch = useAppDispatch();
   const { activeLocale } = useLocalization();
+  const { availableProjects: projects, getProjectName } = useProjects();
+  const { isAcceleratorRoute } = useAcceleratorConsole();
 
   const [deliverables, setDeliverables] = useState<ListDeliverablesElement[]>([]);
   const [deliverablesSearchRequestId, setDeliverablesSearchRequestId] = useState('');
   const deliverablesSearchRequest = useAppSelector(selectDeliverablesSearchRequest(deliverablesSearchRequestId));
 
-  const featuredFilters: FilterConfig[] = useMemo(
-    () =>
-      activeLocale
-        ? [
-            {
-              field: 'status',
-              // These options are strings for now, but may end up as enums when the BE types come through, if that is
-              // the case we will need to implement the renderOption and pillValueRenderer to render the desired
-              // human readable values
-              options: DeliverableStatuses,
-              label: strings.STATUS,
-            },
-            {
-              field: 'category',
-              // Same note as above
-              options: DeliverableCategories,
-              label: strings.CATEGORY,
-            },
-          ]
-        : [],
-    [activeLocale]
-  );
+  const featuredFilters: FilterConfig[] = useMemo(() => {
+    const filters: FilterConfig[] = [
+      {
+        field: 'status',
+        // These options are strings for now, but may end up as enums when the BE types come through, if that is
+        // the case we will need to implement the renderOption and pillValueRenderer to render the desired
+        // human readable values
+        options: DeliverableStatuses,
+        label: strings.STATUS,
+      },
+      {
+        field: 'category',
+        // Same note as above
+        options: DeliverableCategories,
+        label: strings.CATEGORY,
+      },
+    ];
+
+    // show the project filter only in the accelerator route
+    // the participant view already has a projects filter above the table
+    if (isAcceleratorRoute) {
```

Comment:
These components should be "view unaware". Can you pass this filter in as another prop?